tests/functional/v1/test_leasing_positive.py

```diff
@@ -0,0 +1,49 @@
+import tests.functional.base as base
+
+class PositiveLeasingTests(base.EsiFunctionalBase):
+    @classmethod
+    def setUpClass(cls):
+        super(PositiveLeasingTests, cls).setUpClass()
+        cls._init_dummy_cloud(cls, 'owner', 'owner')
+        cls._init_dummy_cloud(cls, 'lessee', 'lessee')
+
+        cls._init_client(cls, 'owner')
+        cls._init_client(cls, 'lessee')
+
+        cls.dummy_node = cls._new_dummy_node(cls, 'owner')
+
+    def setUp(self):
+        super(PositiveLeasingTests, self).setUp()
+        self.clients = PositiveLeasingTests.clients
+
+    def test_offer_self(self):
```

Comment:
Maybe 'test_offer_base' or something? '_self' is a little vague

Reply:
Figuring out the semantics of how these tests are named, ordered, structured, etc. is still very much a WIP-- will work on this though.